src/pages/docs/deployments/custom-scripts/run-a-script-step.md

```diff
@@ -56,6 +56,12 @@ You may also select the source of the script, either:
 Using scripts from inside a package or a git repository are a great way to version and source control your scripts. (You can be assured the correct version of your script will be run when deploying each version of your application.) Both methods have benefits and suit different applications: choose the method best suited to your situation.
 :::
 
+:::div{.hint}
+From Octopus **2024.1**, if you are using [version control](/docs/projects/version-control) with your project you can source files from the same Git repository as your deployment process by selecting Project as the Git repository source. When creating a Release, the commit hash used for your deployment process will also be used to source the files.
```

Comment:
I was trying to move away from using 'version control' and more to just using 'Git' as much as possible here. So this would be something along the lines of: '...if your project configuration is stored in a Git repository...'.

I'm not sure if it's worth changing this at this point, because this will be far from the only place where we're using version control, but it would be good to have some stronger rules. If / when we eventually do that, we can do a sweep of all the documentation and fix it all up.

Reply:
Yeah it's a tricky one, I used "version control" because that was the name of the folder, but looking at the content it's actually mostly called "Configuration as code". Maybe I'll readjust to similar to what you have but have the link say something like  "using the Configuration as code feature" so it's a bit more consistent.

Reply:
I think configuration as code is appropriate in documentation. I've been trying to avoid it in the product as much as possible because it's more of a marketing term rather than a product term, but I think it's safe to have it in the documentation.